### DISCUSSION TITLE
[PSD2] Exercise 1.29 (variant)

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#125] [Esercizi (A-L)]
**Exercise 1.29 (variant)** Convert the following decimal numbers to 8-bit 2’s complement

(a) $42_{10}$
(b) $−63_{10}$
(c) $124_{10}$
(d) $−128_{10}$
(e) $133_{10}$

- _**N.B.:** I numeri da convertire sono gli stessi dell'Esercizio 1.29 originario (#122 ) ma questa volta sono da convertire in rappresentazione in complemento a due anziché in rappresentazione sign/magnitude._

**Comment by Elia-Belli** [comment]
Da Decimale a Ca2 (8bit)

(a) $42_{10} = 00101010_2$
(b) $-63_{10} = 11000001_2$
(c) $124_{10} = 01111100_2$
(d) $-128_{10} = 10000000_2$
(e) $133_{10} = 010000101_2$ (overflow)
